
# Success or failure due to inconsistent views
int32 from_drone_id
int32 to_drone_id
int32 status 
float64 stamp